### PR TITLE
Fix: Refactored the code and fixed Help Page Layout 

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

In this PR i have refactored some code in the Help Page and added jsDocs 

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: src/packages/@app/styles/style.scss [hunk 1]
```diff
@@ -106,6 +106,9 @@ $theme-grey-808080: #808080;
 $theme-grey-636566: #636566;
 $theme-grey-60616B: #60616b;
 $theme-grey-484848: #484848;
+$theme-grey-4D4D4D: #4D4D4D;
+$theme-grey-45494D: #45494D;
+$theme-grey-4e4e4e: #4e4e4e;
 $theme-grey-4D4D4D: #4d4d4d;
```
Comment (itsmdasifraza) [inline src/packages/@app/styles/style.scss:112]: duplicates

File: src/packages/@support/common/components/HelpInfo-Card/HelpInfoCard.svelte [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+<div>
+  {#each status as status}
+    <div class="p-2 review-card">
+      <div class=" d-flex justify-content-between align-items-center">
+        <div>
+          <p class="card-title pb-0 mb-0" style="font-weight: 500; ">
+            {status.title}
+          </p>
+          <span class="category">{status?.category?.name}</span>
+        </div>
```
Comment (itsmdasifraza) [inline src/packages/@support/common/components/HelpInfo-Card/HelpInfoCard.svelte:16]: add ellipse class here to wrap overflow

File: src/packages/@support/features/roadmap/layout/Roadmap.svelte [hunk 3]
```diff
@@ -0,0 +1,232 @@
+<script>
+  import { SearchIcon } from "$lib/assets/icons";
+  import { Select } from "@library/forms";
```
Comment (itsmdasifraza) [inline src/packages/@support/features/roadmap/layout/Roadmap.svelte:3]: import this from library only

File: src/packages/@support/features/roadmap/layout/Roadmap.svelte [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+
+    posts.forEach((post) => {
+      let mappedStatus = post.status;
+      // Check if mappedStatus exists in statusMap, if not, use it as it is
+      if (statusMap[mappedStatus]) {
+        statusMap[mappedStatus].products.push(post);
+      }
+    });
+
+    // Convert the statusMap object to an array suitable for rendering in Svelte
```
Comment (itsmdasifraza) [inline src/packages/@support/features/roadmap/layout/Roadmap.svelte:36]: jsdoc comments

File: src/packages/@support/features/release-notes/layout/ReleaseNotes.svelte [hunk 5]
```diff
@@ -0,0 +1,453 @@
+<script>
+  import { SearchIcon } from "$lib/assets/icons";
+  import { Select } from "@library/forms";
```
Comment (itsmdasifraza) [inline src/packages/@support/features/release-notes/layout/ReleaseNotes.svelte:3]: import it from library

File: src/packages/@support/features/release-notes/layout/ReleaseNotes.svelte [hunk 6]
```diff
@@ -0,0 +1,453 @@
...
+      reactions: 41,
+    },
+    {
+      date: "Jul 31, 2026",
+      title: "v2.4.4 - Latest Version",
+      description:
+        "The latest update includes improved JSON Schema Validation. This enhancement ensures more accurate validation of complex JSON structures, helping you catch errors early in the testing process. The latest update includes improved JSON Schema Validation. This enhancement ensures more accurate validation of complex JSON structures, helping you catch errors early in the testing process",
+      tags: ["New"], // Example with only one tag
+      link: "Github",
+      reactions: 41,
```
Comment (itsmdasifraza) [inline src/packages/@support/features/release-notes/layout/ReleaseNotes.svelte:39]: not required

File: src/packages/@support/features/my-activity/layout/MyActivity.svelte [hunk 7]
```diff
@@ -0,0 +1,764 @@
...
+  user.subscribe((value) => {
+    userInfo = value;
+  });
+
+  let id = "";
+  const getPosts = async (sortType) => {
+    currentSort = sortType;
+
+    console.log(userInfo._id);
+    const listPosts = await fetchPosts(currentSort, userInfo._id);
```
Comment (itsmdasifraza) [inline src/packages/@support/features/my-activity/layout/MyActivity.svelte:47]: not required

Comment (Abhishek9503) [reply src/packages/@support/features/my-activity/layout/MyActivity.svelte:47]: Resolved

File: src/packages/@app/styles/style.scss (hunk 1 above)
Comment (Abhishek9503) [reply src/packages/@app/styles/style.scss:112]: Resolved

File: src/packages/@support/common/components/HelpInfo-Card/HelpInfoCard.svelte (hunk 2 above)
Comment (Abhishek9503) [reply src/packages/@support/common/components/HelpInfo-Card/HelpInfoCard.svelte:16]: Resolved

File: src/packages/@support/features/roadmap/layout/Roadmap.svelte (hunk 4 above)
Comment (Abhishek9503) [reply src/packages/@support/features/roadmap/layout/Roadmap.svelte:36]: Resovled

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Review by Astitva877 — APPROVED** [review]

**Pull request merged**